### ISSUE TITLE
[🐞] Production build downloads additional ~50kB

### ISSUE THREAD

**Issue by Varixo** [BuilderIO/qwik#3756]
### Which component is affected?

Qwik Runtime

### Describe the bug

Every project built with Qwik downloads 2 additional files (~50kB), even if it only contains simple text.

![obraz](https://user-images.githubusercontent.com/78650133/231566560-6c197d83-5f82-4420-9ee5-f61444ff9db7.png)

### Reproduction

https://stackblitz.com/edit/qwik-starter-8kv66t?file=src/routes/index.tsx

### Steps to reproduce

1. Download stackblitz project
2. Build and run
3. Check network tab

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (16) x64 AMD Ryzen 7 5800X 8-Core Processor
    Memory: 18.29 GB / 31.93 GB
  Binaries:
    Node: 18.12.1 - ~\AppData\Local\pnpm\node.EXE
    npm: 8.19.2 - ~\AppData\Local\pnpm\npm.CMD
  Browsers:
    Edge: Spartan (44.22621.1413.0), Chromium (112.0.1722.39)
    Internet Explorer: 11.0.22621.1
```

**Comment by Dindaleon** [comment]
yes, this is a known issue:
https://github.com/BuilderIO/qwik/issues/2998
https://github.com/BuilderIO/qwik/issues/3298
https://github.com/BuilderIO/qwik/issues/3390

**Comment by Dindaleon** [comment]
Sorry to bump but I would like to know if there has been any progress on this.

**Comment by Varixo** [comment]
For this case possible fix is in 0.105.0 version. After the update, only 260B is downloaded and that's fine with me.

![obraz](https://user-images.githubusercontent.com/78650133/235119269-1e0a67c9-69fd-41ed-9257-afe198fbfe95.png)

**Comment by Varixo** [comment]
yep, for more interactive websites some additional files are still downloaded. There are two more opened issues for this, so I think this can stay closed.
